src/controllers/Lane.js

```diff
@@ -19,15 +19,16 @@ class Lane extends Component {
     currentPage: this.props.currentPage,
     addCardMode: false,
     collapsed: false,
-    isDraggingOver: false
+    isDraggingOver: false,
+    lastPage: false
```

Comment:
We have currentPage as number. Probable it will be better to rename this prop to isPageLast or to give total pages count in pros like 'totalPages'

Reply:
Yes, it will be better to rename to isPageLast. Moreover you do not have to extend your API handle onLaneScroll by new 'totalPages' after upgrade.